interface/new/new_comprehensive_save.php

```diff
@@ -87,7 +87,7 @@
   }
   $picture_url = "";
   //begin file uploading
-  $destination_directory = "../../profile_pictures/";
+  $destination_directory = "../../sites/".$_SESSION['site_id']."/profile_pictures/";
```

Comment:
$GLOBALS['OE_SITES_BASE'] = "$webserver_root/sites";
Use the above global to hit the sites directory instead of a relative dir.